test/imported.test.js

```diff
@@ -6,7 +6,7 @@
 'use strict';
 
 /* eslint-env node, mocha */
-describe('db2 imported features', function() {
+describe('dashdb imported features', function() {
```

Comment:
They are. But in each module they are called with the `module name` imported features. For example [mysql](https://github.com/strongloop/loopback-connector-mysql/blob/master/test/imported.test.js#L7)